src/node_file.cc

```diff
@@ -190,15 +380,15 @@ bool FSReqAfterScope::Proceed() {
 }
 
 void AfterNoArgs(uv_fs_t* req) {
-  FSReqWrap* req_wrap = static_cast<FSReqWrap*>(req->data);
+  FSReqBase* req_wrap = static_cast<FSReqBase*>(req->data);
```

Comment:
Side note (maybe try to remind me once this lands): There should be a `ReqWrap<T>::From(T* uv_request)` method that a) uses `ContainerOf` instead of requiring the lookup and b) doesn’t rely on blindlly casting void pointers to C++ non-base classes, because the latter is bound to go wrong at some point